descriptions/css/properties/border-left.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "border-left": {
+        "__short_description": "The <strong><code>border-left</code></strong> <a href='https://developer.mozilla.org/docs/Web/CSS'>CSS</a> property is a shorthand that sets the values of <a href='https://developer.mozilla.org/docs/Web/CSS/border-left-width'><code>border-left-width</code></a>, <a href='https://developer.mozilla.org/docs/Web/CSS/border-left-style'><code>border-left-style</code></a> and <a href='https://developer.mozilla.org/docs/Web/CSS/border-left-color'><code>border-left-color</code></a>."
+      }
```

Comment:
Super-nit: instead of "The border-left CSS property is a shorthand that sets", "The border-left shorthand CSS property sets" is shorter and (I think) more readable, and is the same style as for animation (https://developer.mozilla.org/en-US/docs/Web/CSS/animation).

Reply:
Good idea on this. I've added a column to my spreadsheet to review this for consistency